gems/aws-sdk-s3/spec/encryption/client_spec.rb

```diff
@@ -632,31 +632,29 @@ def stub_encrypted_get_with_instruction_file(sfx = '.instruction')
             "\x8E\x0E\xC0\xD5\x1A\x88\xAF2\xB1\xEEg#\x15"
           end
 
-          if RUBY_VERSION > '1.9.3' && OpenSSL::Cipher.ciphers.include?('aes-256-gcm')
-            it 'supports decryption via KMS w/ GCM' do
```

Comment:
We still need to include the second check here AFAIK - there are instances where the openssl library locally does not support that this this that are outside of ruby version.